.github/workflows/docker-ingestion.yml

```diff
@@ -3,21 +3,26 @@ on:
   push:
     branches:
       - master
-    paths-ignore:
-      - "docs/**"
-      - "**.md"
+    paths:
+      - "metadata-ingestion/**"
```

Comment:
I think we want to build the core containers on each commit?

Reply:
why do we need to push it every time? We only use head in quickstart so when nothing is changing why do we need to build it every time?